product_uom_measure_type/models/product_template.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (C) 2021 - Today: GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    uom_measure_type = fields.Selection(
+        string="UoM Measure Type",
+        related="uom_id.category_id.measure_type", store=True)
```

Comment:
Does this double dotted introduce a perf penalty ?

Reply:
I really don't know ! 

Well, checking again, I saw that measure_type was present in uom.uom model. (related, store=True). So, changed for a single dotted relation.